source/tutorials/table-filtering.rst

```diff
@@ -0,0 +1,88 @@
+
+.. command-block::
+
+    curl -sL "https://docs.google.com/spreadsheets/d/1_3ZbqCtAYx-9BJYHoWlICkVJ4W_QGMfJRPLedt_0hws/export?gid=0&format=tsv" > sample-metadata.tsv
+    curl -sLO https://data.qiime2.org/2.0.6/tutorials/filtering-feature-tables/table.qza
+
+Frequency-based filtering
+-------------------------
+
+Frequency-based filtering is used to filter samples or features based on their total frequency.
+
+When filtering samples this can be used, for example, to filter samples whose total frequency is an outlier in the distribution of sample frequencies. In many 16S surveys, only a few (perhaps tens) of sequences will be obtained for some samples, possibly due to low biomass of the sample resulting in low DNA extraction yield. In this case, the user may want to remove samples based on their minimum total frequency (i.e., total number of sequences obtained for the sample, in this example). This can be achieved as follows (in this example, samples with a total frequency less than 1500 will be filtered).
+
+.. command-block::
+    qiime feature-table filter-samples --i-table table.qza --o-filtered-table filtered-table --p-min-frequency 1500
+
+This filter can be applied to the feature axis to remove low abundance features from a table. For example, you can remove all features with a total abundance (summed across all samples) of less than 10 as follows.
+
+.. command-block::
+    qiime feature-table filter-features --i-table table.qza --o-filtered-table filtered-table --p-min-frequency 10
+
+Both of these methods can also be applied to filter based on the maximum total frequency using the ``--p-max-frequency``. The ``--p-min-frequency`` and ``--p-max-frequency`` can be combined to filter based on lower and upper limits of frequency.
+
+Contingency-based filtering
+---------------------------
+
+Contingency-based filtering is used to filter samples from a table contingent on the number of features they contain, or to filter features from a table contingent on the number of samples they're observed in.
+
+This filtering is commonly used for filtering features that show up in only one or a few samples, based on the suspicion that these may not represent real biological diversity but rather experimental artifacts such as PCR chimeras. Features that are present in only a single sample could be filtered from a feature table as follows.
+
+.. command-block::
+    qiime feature-table filter-features --i-table table.qza --o-filtered-table filtered-table --p-min-samples 2
+
+Similarly, samples that contain only a few features could be filtered from a feature table as follows.
+
+.. command-block::
+    qiime feature-table filter-samples --i-table table.qza --o-filtered-table filtered-table --p-min-features 10
+
+Both of these methods can also be applied to filter contingent on the maximum number of features or samples, using the ``--p-max-features`` and ``--p-max-samples`` parameters.
+
+Identifier-based filtering
```

Comment:
Went with *Index-based*, and indicated that this refers to identifiers. In QIIME 1 we usually refer to these as *identifiers*, but I think we should transition to this terminology (which will important for consistency when we have real index/metadata support).